A/Atomsk/build_tarballs.jl

```diff
@@ -3,29 +3,45 @@
 using BinaryBuilder, Pkg
 
 name = "Atomsk"
-version = v"0.11.2"
+version = v"0.12.0"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/pierrehirel/atomsk.git", "3333858281e0ebd6279825b83ab871cf4d050a8d")
+    GitSource("https://github.com/byu-cxi/atomsk.git", "adeb5d5c059a16bf649ab6657cc550b5868fac0b"),
```

Comment:
Why using a fork?

Reply:
The maintainer is currently busy and hasn't had time to look over my pull request. So I put mine temporarily until he has time to check things out.

Reply:
Everything is pushed up now.